telegram/ext/_conversationhandler.py

```diff
@@ -304,7 +320,7 @@ def __init__(
         self._name = name
         self._map_to_parent = map_to_parent
 
-        self.timeout_jobs: Dict[Tuple[int, ...], 'Job'] = {}
+        self.timeout_jobs: Dict[Tuple[int, ...], 'Job'] = {}  # TODO: figure out purpose of this
```

Comment:
it keeps track of the jobs that are scheduled for conversation timeouts so we can abort them if a new update comes in in time

Reply:
okay, so as I understand it, whenever a new update comes in, the previous job is cancelled and removed, and the new one is added in this dict. So shouldn't the max size be 1? Meaning it doesn't have to be a dict?

Reply:
well, for each conversation key (e.g. for each user), there is a different job